vm/src/types/program.rs

```diff
@@ -136,6 +136,24 @@ impl Program {
         self.shared_program_data.identifiers.get(id)
     }
 
+    pub fn get_relocated_instruction_locations(
+        &self,
+        relocation_table: Vec<usize>,
```

Comment:
We could probably take a `&[usize]` instead. That is both more flexible and saves us from cloning to pass ownership.

Reply:
Fixed